Workbooks/Virtual Machines - Performance Analysis/Performance Analysis for a Group of VMs/Performance Analysis for a Group of VMs.workbook

```diff
@@ -284,47 +240,58 @@
             "version": "KqlParameterItem/1.0",
             "name": "Test",
             "type": 1,
-            "isRequired": false,
-            "query": "// {DefaultWorkspace} {ContextFree} {ContextSelection} {RandomSelection} {Selection} {Subscriptions} {Workspaces}\r\nPerf\r\n| take 1",
+            "query": "Perf\r\n| where TimeGenerated {TimeRange}\r\n| take 1",
             "crossComponentResources": [
               "{Workspaces}"
             ],
             "isHiddenWhenLocked": true,
-            "timeContextFromParameter": null,
             "queryType": 0,
             "resourceType": "microsoft.operationalinsights/workspaces"
           }
         ],
         "style": "above",
-        "queryType": 1,
-        "resourceType": "microsoft.resourcegraph/resources"
+        "queryType": 0,
+        "resourceType": "microsoft.insights/components"
       },
-      "conditionalVisibility": null
+      "name": "parameters - 2"
     },
     {
       "type": 1,
       "content": {
-        "json": "⚠ This workbook requires [Azure Monitor for VMs (preview)](https://docs.microsoft.com/en-us/azure/azure-monitor/insights/vminsights-overview) enabled in order to show accurate and complete information for your VMs. Follow the link to onboard machines or select a different workspace."
+        "json": "🛑 There was an error initializing this workbook, select an item under the `Subscriptions` dropdown or refresh the workbook."
```

Comment:
nit: I personally wouldn't callout "there was an error", but advising them to select a subscription is a good check

Reply:
Okay, I have updated the user prompt as requested.